From 25b37b4c8c8c5aef7cd530d919258bd8e5dce068 Mon Sep 17 00:00:00 2001
From: Xisco Fauli <xiscofauli@libreoffice.org>
Date: Wed, 3 Jul 2024 14:08:47 +0200
Subject: [PATCH] raptor: Use --without-www

it seems not used. Besides, it fails when upgrading
to libxml2 2.13.
See https://gerrit.libreoffice.org/c/core/+/169327

Change-Id: If383130eac3b5d6de911c4c273c8e371a4980f23
Reviewed-on: https://gerrit.libreoffice.org/c/core/+/169933
Tested-by: Jenkins
Reviewed-by: Michael Stahl <michael.stahl@allotropia.de>
Reviewed-by: Xisco Fauli <xiscofauli@libreoffice.org>
---
 external/redland/ExternalProject_raptor.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/external/redland/ExternalProject_raptor.mk b/external/redland/ExternalProject_raptor.mk
index 8ac05880766b..192bbade94e7 100644
--- a/external/redland/ExternalProject_raptor.mk
+++ b/external/redland/ExternalProject_raptor.mk
@@ -29,8 +29,8 @@ $(call gb_ExternalProject_get_state_target,raptor,build):
 			$(if $(SYSBASE),$(if $(filter LINUX SOLARIS,$(OS)),-L$(SYSBASE)/lib -L$(SYSBASE)/usr/lib -lpthread -ldl))" \
 		CPPFLAGS="$(if $(SYSBASE),-I$(SYSBASE)/usr/include) $(gb_EMSCRIPTEN_CPPFLAGS)" \
 		$(gb_RUN_CONFIGURE) ./configure --disable-gtk-doc \
-			 --enable-parsers="rdfxml ntriples turtle trig guess rss-tag-soup" \
-			--with-www=xml \
+			--enable-parsers="rdfxml ntriples turtle trig guess rss-tag-soup" \
+			--without-www \
 			--without-xslt-config \
 			$(gb_CONFIGURE_PLATFORMS) \
 			$(if $(CROSS_COMPILING),$(if $(filter INTEL ARM,$(CPUNAME)),ac_cv_c_bigendian=no)) \
-- 
2.47.0

